### ISSUE TITLE
Keyboard focus indicator contrast is less than 3:1 on More Actions list items and context menu items in Visual Studio Dark and Light 2026 themes:A11y_VisualStudioCode_Non text Contrast

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#314929]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier." 

"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".

## GitHub Tags: 
#A11yTCS;#Visual Studio Code Client; #E2E_VisualStudioCodeClient_Win32_Dec2025;#DesktopApp;#FTP;#Win32;#A11ySev2;#A11yMAS;#WCAG1.4.11; #NonText-Contrast;  #Closed;  #A11yFixed;

## Environmental Details: 
App Name: Visual Studio Code Client 
Version 1.119.0
Microsoft Windows 11 Enterprise 
OS Build:  26200.7392

## Repro Steps: 
1. Open Visual Studio Code application
2. TAB to Azure in left navigation and press ENTER key.
3. TAB to More action and press down arrow key or TAB to any item and open context menu and press down arrow key
4. Apply Dark 2026 and light 2026 theme
7. Observe that keyboard indicator color contrast is less than 3:1

## Actual Results: 
In Visual Studio Dark 2026 and Light 2026 themes, the keyboard focus indicator on More Actions list items and context menu items has color contrast less than 3:1 against adjacent background colors, making it difficult to visually identify the focused element.

## Expected Results: 
The keyboard focus indicator should have a minimum contrast ratio of 3:1 against adjacent colors so that it is clearly visible.

## User Impact: 
Low vision users may not be able to identify the focused item.
Keyboard-only users may lose track of current focus, affecting navigation and interaction.
Leads to reduced usability and accessibility compliance

### Light 2026 theme

<img width="1279" height="555" alt="Image" src="https://github.com/user-attachments/assets/dbecc90d-a7ce-4ca4-b50a-71361a27b31e" />

<img width="1276" height="552" alt="Image" src="https://github.com/user-attachments/assets/21e01b02-e5e1-44ad-9304-4385f50fa36e" />

### Dark 2026 theme

<img width="1279" height="550" alt="Image" src="https://github.com/user-attachments/assets/0fe85351-b4ca-4a8f-ae85-6993d6a85d35" />

<img width="1279" height="557" alt="Image" src="https://github.com/user-attachments/assets/744d1f72-2679-4290-9b42-63c21405a72f" />

**Comment by kapilvaishna** [comment]
Issue is not repro in VS code insider 1.124.0 In Light 2026 and dark 2026 theme so closing the bug.

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/5186c7b4-01f0-4200-9397-8549072ee058" />


<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/5c1fb300-87c5-495d-bbd5-9570b2926f67" />


<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/8f5ef9b2-3835-40d5-9e4e-650a5d303cce" />

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/b8d9d270-d235-4fda-bc60-3624bac918f7" />
